### DISCUSSION TITLE
Affichage sur plusieurs colonnes pour les applications internes

### DISCUSSION THREAD

**Discussion by jpbeaulieu1** [MTESSDev/utd-webcomponents#91] [Idées]
Bonjour
Il serait intéressant de pouvoir afficher les champs d'un formulaire sur plusieurs colonnes pour les applications internes. Un peu comme la grid de bootstrap.

https://getbootstrap.com/docs/4.0/layout/grid/

Merci

**Comment by j0ber** [comment]
C'est en cours en ce moment :)  je pourrai te fournir un lien bientôt

**Reply by jpbeaulieu1** [reply]
Super Merci https://lively-cliff-018a9540f.azurestaticapps.net/composants/formulaire/grillesysteme
